src/test/java/seedu/address/logic/commands/commandtestutils/FlashcardBankCommandTestUtil.java

```diff
@@ -41,6 +47,8 @@ public static void assertCommandSuccess(Command<FlashcardModel> command, Flashca
         try {
             CommandResult result = command.execute(actualModel);
             assertEquals(expectedCommandResult, result);
+            System.out.println(expectedCommandResult.getFeedbackToUser());
+            System.out.println(result.getFeedbackToUser());
```

Comment:
May I check the reason for the system output here? If it is used for debug, maybe can just delete them.